src/spid-localstorage.js

```diff
@@ -25,7 +25,7 @@ function set(key, value, expiresInSeconds) {
 
 function clear(key) {
```

Comment:
I'm thinking that renaming the method is probably a thing for a separate PR (as that's more an issue of style), if we decide that it's the right thing to do.